Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -461,6 +462,11 @@ void AzAssetBrowserWindow::CreateToolsMenu()
 
     if (ed_useWIPAssetBrowserDesign)
     {
+        auto* openNewAction = new QAction(tr("Open New Instance"), this);
```

Comment:
Could you rename this to `Open another Asset Browser` to keep the same naming as the option from the right-click menu?

Reply:
Done